source/common/stats/stats_impl.cc

```diff
@@ -149,13 +149,55 @@ bool TagExtractorImpl::extractTag(const std::string& stat_name, std::vector<Tag>
   return false;
 }
 
+RawStatData* BlockRawStatDataAllocator::alloc(const std::string& name) {
```

Comment:
I moved a lot of these tests from `hot_restart_impl_test.cc` since they are really unit tests on the allocator itself. A few stayed in `hot_restart_impl_test.cc` as they tested behavior across hot restart.

Reply:
What tests are you referring to? This is an implementation file...?